### PR TITLE
[PLAYER-3702] Added accessibility for playback speed feature

### PR DESCRIPTION
**Pull request by MarkEPAM**
- added accessibility for playback speed feature

### CODE REVIEW

**Review by brol1dev — APPROVED** [review]
LGTM, changing destination branch to `candidate`

File: sdk/react/widgets/controlBarWidgets.js [hunk 1]
```diff
@@ -258,11 +259,18 @@ class controlBarWidget extends React.Component {
 
   playbackSpeedWidget = (options) => {
     let widget = null;
+    
+    // Create accessibility label for selected playback speed rate button
+    
+    const playbackSpeedRateWithoutFostfix = options.selectedPlaybackSpeedRate.slice(0,-1);
```
Comment (pilievOoyala) [inline sdk/react/widgets/controlBarWidgets.js]: There is a typo here: playbackSpeedRateWithoutFostfix -> playbackSpeedRateWithoutPostfix
Should be PostFix not FostFix

**Comment by pilievOoyala** [comment]
LGTM aside from the typo

**Comment by brol1dev** [comment]
@pilievOoyala fixed it myself so we can include it in today's RC.

**Pull request merged**